Makefile.dep

```diff
@@ -687,7 +687,9 @@ ifneq (,$(filter phydat,$(USEMODULE)))
 endif
 
 ifneq (,$(filter evtimer,$(USEMODULE)))
-  USEMODULE += xtimer
+  ifeq (,$(filter evtimer_on_ztimer,$(USEMODULE)))
+    USEMODULE += xtimer
+  endif
```

Comment:
Please move the other dependency handling (see below) here, so that all `evtimer` related dependency management comes directly below `ifneq (,$(filter evtimer,$(USEMODULE)))`